src/components/dialogs/limits/limits-side-pane.tsx

```diff
@@ -129,8 +121,19 @@ const LimitsPane = ({ id = LIMITS, currentNode, equipmentToModify, clearableFiel
         [getValues, temporaryLimitHasPreviousValue]
     );
 
-    const getTemporaryLimitPreviousValue = useCallback(
-        (rowIndex, column, arrayFormName, temporaryLimits) => {
+    const findTemporaryLimit = useCallback(
+        (rowIndex: number, arrayFormName: string, temporaryLimits: TemporaryLimit[]) => {
+            return temporaryLimits?.find(
+                (e: TemporaryLimit) =>
+                    e.name === getValues(arrayFormName)[rowIndex]?.name &&
+                    e.acceptableDuration === getValues(arrayFormName)[rowIndex]?.acceptableDuration
+            );
+        },
+        [getValues]
+    );
+
+    const getPreviousValue = useCallback(
+        (rowIndex: number, column: ILimitColumnDef, arrayFormName: string, temporaryLimits: TemporaryLimit[]) => {
             const formattedTemporaryLimits = formatTemporaryLimits(temporaryLimits);
```

Comment:
you have to check on the temporaryLimits lenght
```suggestion
                       if (!temporaryLimits?.length) {
                return undefined;
            }
```

Reply:
OK done.